build.gradle

```
@@ -8,27 +8,29 @@ plugins {
     id 'maven-publish'
 }
 
-group = 'com.iexec.result-proxy'
-sourceCompatibility = 11
-targetCompatibility = 11
-
 ext {
     springCloudVersion = '2021.0.0'
     gitBranch = 'git rev-parse --abbrev-ref HEAD'.execute().text.trim()
 }
 
-if (gitBranch != 'main' && gitBranch != 'master' && ! (gitBranch ==~ '(release|hotfix|support)/.*')) {
-    version += '-NEXT-SNAPSHOT'
-}
+allprojects {
```

Why not sharing publishing (and jacoco)?
```suggestion

allprojects {
    apply plugin: 'jacoco'
    apply plugin: 'maven-publish'
    
    [...]
```

---

Following our discussions and the following links (see below), we can wait a little bit before tackling this subject.
It will be raised back later when we will:
* Update gradle version
* Move Spring boot projects from root project to a sub-project

Several links used in the discussions:
* https://docs.gradle.org/current/userguide/plugins.html
* https://docs.gradle.org/current/userguide/sharing_build_logic_between_subprojects.html#sharing_build_logic_between_subprojects
* https://docs.gradle.org/current/samples/sample_convention_plugins.html

Project links:
* https://github.com/web3j/web3j/blob/master/build.gradle
* https://github.com/web3j/web3j/blob/master/build.gradle